plugins/module_utils/client_path_helper.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8 -*-
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+import sys
+from pathlib import Path
+
+try:
+    # Adjust the Python path to include the generated client
+    sys.path.append(str(Path(__file__).resolve().parent.parent.parent / "lib/flightctl_api_client"))
```

Comment:
This is a side effect of the client being in `/lib/flightctl_api_client`.

If the client were nested in `module_utils` or another location the absolute import paths inside the generated code don't work.  Two alternatives are to modify the generated code to use different import paths (which involves building custom templating ourselves or some hacks like generating the files in a deeply nested directory then moving it so the import names are constructed as we want) OR modify the import location like this, felt it was better to handle the import of the client rather than focus on modifying the generated code which imo would be more brittle over time.